backend-models/service-tests/codeviewrService-test.txt

```diff
@@ -0,0 +1,214 @@
+Prepare data  Line 71
+—---------------------------------------------------------------------------------
+**********
+*pre-req:*
+**********
+(Test for this is under construction)
+{ (checklogin = true) 
+&& (hasAccess($userid, $courseId, 'w') || hasAccess($userid, $courseId, 'st')) == true} give $writeAccess="w";, else $writeAccess="s";
+
+send{
+$exampleCount=1;
+	$exampleId=9030; 
+	$exampleName=New Code;
+	$courseID = 1885
+	$cversion= 1337;
+	$beforeId= [];
+	$afterId= [];
+	$public=0;
+	$sectionName=New Code9030;
+	$playlink= NULL;
+}
+SETTEMPL  Line 96-159
```

Comment:
The means to test is currently being developed, so the idea is that we merge and fix on a later term so it still is available to others if needed

I made an inset note but not sure if needed. Since this function calls editexample, we decided to exclude the expected output to avoid redundance in the documentation